editor/components/inserter/index.js

```diff
@@ -36,8 +41,12 @@ class Inserter extends wp.element.Component {
 					icon="insert"
 					label={ wp.i18n.__( 'Insert block' ) }
 					onClick={ this.toggle }
-					className="editor-inserter__toggle" />
-				{ opened && <InserterMenu position={ position } onSelect={ this.close } /> }
+					className="editor-inserter__toggle"
+					aria-haspopup="true"
+					id="inserter-toggle"
+					buttonRef={ ( node ) => this.nodes.toggle = node }
```

Comment:
This is to grab the reference for the icon button.  I couldn't figure out another way to do this.  When the menu closes we need to refocus the inserter button for accessibility.  I would definitely like to find something better.